README.md

```diff
@@ -1,128 +1,138 @@
-
-Install [IBM Cloud Developer Tools](https://cloud.ibm.com/docs/cli?topic=cloud-cli-getting-started) on your machine by running the following command:
-```
-curl -sL https://ibm.biz/idt-installer | bash
-```
-
-Create an application on IBM Cloud by running:
-
-```bash
-ibmcloud dev create
-```
-
-This will create and download a starter application with the necessary files needed for local development and deployment.
-
-Your application will be compiled with Docker containers. To compile and run your app, run:
-
-```bash
-ibmcloud dev build
-ibmcloud dev run
-```
-
-This will launch your application locally. When you are ready to deploy to IBM Cloud on Cloud Foundry or Kubernetes, run one of the commands:
-
-```bash
-ibmcloud dev deploy -t buildpack // to Cloud Foundry
-ibmcloud dev deploy -t container // to K8s cluster
-```
-
-You can build and debug your app locally with:
-
-```bash
-ibmcloud dev build --debug
-ibmcloud dev debug
-```
-
-## Next Steps
-* Learn more about the services and capabilities of [IBM Cloud](https://cloud.ibm.com).
-* Explore other [sample applications](https://cloud.ibm.com/developer/appservice/starter-kits) on IBM Cloud.
+# Prometeo Web App
+
+This repository contains the [Prometeo](https://github.com/Code-and-Response/Prometeo) solution web application dashboard for firefighters to view .
+
+This service wakes up every minute and calculates time weighted average exposures for all fire fighters and compares them to the configured limits.
+
+[![License](https://img.shields.io/badge/License-Apache2-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0) [![Slack](https://img.shields.io/badge/Join-Slack-blue)](https://join.slack.com/t/code-and-response/shared_invite/enQtNzkyMDUyODg1NDU5LTdkZDhmMjJkMWI1MDk1ODc2YTc2OTEwZTI4MGI3NDI0NmZmNTg0Zjg5NTVmYzNiNTYzNzRiM2JkZjYzOWIwMWE)
+
+## Contents
+
+- [Prerequisites](#prerequisites)
+- [Run locally with Python](#run-locally-with-python)
+- [Run locally with Docker](#run-locally-with-docker)
+- [Run on Kubernetes](#run-on-kubernetes)
+- [Troubleshooting](#troubleshooting)
+- [Built with](#built-with)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Prerequisites
+1. [Docker](https://docs.docker.com/desktop/)
+2. [IBM CLI](https://cloud.ibm.com/docs/cli?topic=cli-install-ibmcloud-cli)
+3. [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+4. [Helm](https://helm.sh/docs/intro/install/)
+5. [Skaffold](https://skaffold.dev/docs/install/)
+
+## Run locally with Python
+You can run this solution locally in docker as follows
+
+1. Set up environment variables in the `.env` file
+2. Install mariadb locally
+   1. build 
+    ```
+        docker pull mariadb
+    ```
+   2. run the image
+    ```
+        docker run -p 3306:3306 --name prometeo-mariadb -e MYSQL_ROOT_PASSWORD='' -d mariadb
+    ```
+    3. Use the prometeo.sql and procedure.sql to dump into the new database to get the initial framework set. 
+    4. Test the image - TBD
```

Comment:
Once the container is running, they can use the following command to test it out. Might be also useful for loading the dump

```docker exec -it prometeo-mariadb mysql -uroot -p```